src/Compilers/RealParserTests/RandomRealParserTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
I hate to be a spoilsport on this, but isn't the search space so huge that generating random tests is pointless? I mean, assuming we could do a million tests a second, how long would it take before we searched the entire range? (Back-of-an-envelope calculation) Would System.Random wrap around before we got there?
Obviously, if this technique finds just one failure, then it's a success, but how long would we have to run it before the _lack_ of any asserts gave us any confidence in the algorithm?